library/ws-widget/collection/src/lib/_common/content-toc/app-toc-about/app-toc-about.component.html

```diff
@@ -1,3 +1,72 @@
+<ng-template #certificateCard>
+    <div class="certificate-outer-div">
+        <div class="certificate-card">
+            <div class="flex flex-col certificate-info p-4 gap-3">
+                <div class="flex flex-row">
+                    <div class="flex-1">
+                        <ng-container *ngIf="skeletonLoader">
+                            <ws-widget-skeleton-loader [bindingClass]="'flex rounded'" [width]="'200px'"
+                                [height]="'24px'"></ws-widget-skeleton-loader>
+                        </ng-container>
+                        <ng-container *ngIf="!skeletonLoader">
+                            <div class="text-sm font-semibold cursor-pointer">{{ content?.name }}</div>
+                            <div class="text-xs font-light">{{ 'apptocsinglepage.completedCourse' | translate }} {{ content?.completedOn ?
+                                (content?.completedOn | date: 'd MMM, y') : 'NA' }}</div>
+                        </ng-container>
+                    </div>
+                    <div class="flex justify-center">
+                        <ng-container *ngIf="skeletonLoader">
+                            <ws-widget-skeleton-loader [bindingClass]="'flex rounded'" [width]="'24px'"
+                                [height]="'24px'"></ws-widget-skeleton-loader>
+                        </ng-container>
+                        <ng-container *ngIf="!skeletonLoader && content?.certificateObj?.certData">
+                            <mat-icon class="icon cursor-pointer"
+                                (click)="handleOpenCertificateDialog()">arrow_downward</mat-icon>
+                        </ng-container>
+                    </div>
+                </div>
+                <div class="flex-1">
+                    <ng-container *ngIf="skeletonLoader">
+                        <ws-widget-skeleton-loader [bindingClass]="'flex rounded'" [width]="'280px'"
+                            [height]="'148px'"></ws-widget-skeleton-loader>
+                    </ng-container>
+                    <ng-container *ngIf="!skeletonLoader">
+                        <div class="certificate_box" *ngIf="!content?.certificateObj?.certData">
+                            <img src="/assets/icons/toc/no-certificate.svg" alt="No certificate image" />
+                            <div class="flex items-center certificate_overlay">
+                                <div class="flex flex-col text-center gap-1 p-4 text-white mt-4">
+                                    <div>{{'apptochome.certificationTakesTime' | translate }}</div>
+                                    <div>{{'apptochome.issuePersistsMailus' | translate }} mission.karmayogi@gov.in
```

Comment:
can you please remove this
mission.karmayogi@gov.in

Reply:
That's the email where the user will send an email. It is required! And it's mentioned in the design as well